src/tools/miri/src/alloc_addresses/mod.rs

```diff
@@ -141,7 +141,11 @@ trait EvalContextExtPriv<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
         }
     }
 
-    fn addr_from_alloc_id(&self, alloc_id: AllocId) -> InterpResult<'tcx, u64> {
+    fn addr_from_alloc_id(
+        &self,
+        alloc_id: AllocId,
+        _kind: MemoryKind<MiriMemoryKind>,
```

Comment:
The kind isn't used yet here, but the plan is to use it in the future (to avoid reusing addresses of stack allocations).